go/libkb/merkle_client.go

```diff
@@ -1175,6 +1175,7 @@ func (mr *MerkleRoot) ToSigJSON() (ret *jsonw.Wrapper) {
 	ret.SetKey("seqno", jsonw.NewInt(int(*mr.Seqno())))
 	ret.SetKey("ctime", jsonw.NewInt64(int64(mr.Ctime())))
 	ret.SetKey("hash", jsonw.NewString(mr.RootHash().String()))
+	ret.SetKey("hash_meta", jsonw.NewString(mr.ShortHash().String()))
```

Comment:
That's what I thought, for the transition period. If both are provided, both are checked by the server.